packages/core-v2/src/visual-model/default-visual-model.ts

```diff
@@ -387,6 +394,40 @@ export class DefaultVisualModel implements WritableVisualModel, EntityEventListe
     }
   }
 
+  private addToRepresentedEntites(
+    representedToEntityMap: Map<string, string[]>,
```

Comment:
This first argument seems to be alwayes this.representedToEntity.
Why not use it directly from this?

Reply:
Perhaps a change to introduce similar addToMapArray function here.